helix-term/src/commands.rs

```diff
@@ -1399,26 +1399,26 @@ pub fn scroll(cx: &mut Context, offset: usize, direction: Direction) {
 }
 
 fn page_up(cx: &mut Context) {
-    let view = view!(cx.editor);
-    let offset = view.inner_area().height as usize;
+    let (view, _doc) = current!(cx.editor);
+    let offset = view.inner_height();
     scroll(cx, offset, Direction::Backward);
 }
 
 fn page_down(cx: &mut Context) {
-    let view = view!(cx.editor);
-    let offset = view.inner_area().height as usize;
+    let (view, _doc) = current!(cx.editor);
+    let offset = view.inner_height();
     scroll(cx, offset, Direction::Forward);
 }
 
 fn half_page_up(cx: &mut Context) {
-    let view = view!(cx.editor);
-    let offset = view.inner_area().height as usize / 2;
+    let (view, _doc) = current!(cx.editor);
+    let offset = view.inner_height() / 2;
     scroll(cx, offset, Direction::Backward);
 }
 
 fn half_page_down(cx: &mut Context) {
-    let view = view!(cx.editor);
-    let offset = view.inner_area().height as usize / 2;
+    let (view, _doc) = current!(cx.editor);
+    let offset = view.inner_height() / 2;
```

Comment:
These can all be reset to `let view = view!(cx.editor)`

Reply:
Thanks - good catch. Apologies for the sloppiness here. It's been a while since I kicked off the work and the details escape me now.